speculos/mcu/seproxyhal.py

```diff
@@ -243,7 +244,8 @@ def __init__(self,
                  automation_server: Optional[AutomationServer] = None,
                  transport: str = 'hid',
                  fonts_path: str = None,
-                 api_level=None):
+                 api_level=None,
+                 model=None):
```

Comment:
Normally at this point, `main.py` made all the necessary checks and `model` will not be `None`, and `SeProxyHal` is instantiated with all its arguments, so I guess there is no much point using a keyword argument with a default type (`None`) different from the expected type (`str`). Moreover this `None` affectation will need to be checked in `OCR` [here](https://github.com/LedgerHQ/speculos/pull/408/files#diff-2c2fdf03b51cff00e657307577a1f80c4d45f2c331a027ee04c943e40cd6712aR151-R152), whereas we could just trust ourselves that `model` is a `str`.

So let's just assume it's a `model: str` and be done with it. You could just do `model: str = ""` if you _really_ want to keep the keyword argument.